qutebrowser/config/websettings.py

```diff
@@ -177,6 +178,17 @@ def set_default_text_encoding(self, encoding: str) -> bool:
         self._settings.setDefaultTextEncoding(encoding)
         return old_value != encoding
 
+    def set_unknown_url_scheme_policy(self, policy: int) -> bool:
+        """Set the UnknownUrlSchemePolicy to use.
+
+        Return:
+            True if there was a change, False otherwise.
+        """
+        assert policy is not usertypes.UNSET  # type: ignore
+        old_value = self._settings.unknownUrlSchemePolicy()
+        self._settings.setUnknownUrlSchemePolicy(policy)
+        return old_value != policy
+
```

Comment:
This should go to `webenginesettings.py` instead, as it's QtWebEngine-specific code.